js/scripts.js

```diff
@@ -1 +1,10 @@
-﻿
\ No newline at end of file
+﻿var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'Triceratops'.toUpperCase();
+console.log(dinosaur);
+
+
+console.log(text.replace('Velociraptor', dinosaur));
+
+var newText = text.replace('Velociraptor', dinosaur)
```

Comment:
Zrób tak żeby   zamiast "Velociraptor ", zamieniany był zawsze pierwszy wyraz   zmiennej `text` - bez względu na to co tam będzie. Możesz użyć do tego metody   `split` (google -> "mdn split")